### PR TITLE
Change landing messages for uninhabited worlds

### PR DESCRIPTION
**Pull request by TomGoodIdea**
## Summary
When you try to land on a non-landable planet or moon within sun's green zone, it tells you the planet/moon is uninhabited. But there are lots of uninhabited planets you can land on, so it shouldn't be a problem.

I reworded two standard landing messages and now they say there's no place suitable for landing.

## Additional notes
Of course, there's a question why on the whole planet there isn't even a single larger grassland, but I think that's the difference between landable and non-landable planets.

### CODE REVIEW

File: source/System.cpp [hunk 1]
```diff
@@ -421,10 +421,10 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 		static const string STAR = "You cannot land on a star!";
 		static const string HOTPLANET = "This planet is too hot to land on.";
 		static const string COLDPLANET = "This planet is too cold to land on.";
-		static const string UNINHABITEDPLANET = "This planet is uninhabited.";
+		static const string UNINHABITEDPLANET = "This planet doesn't have any place suiatble for landing.";
```
Comment (bene-dictator) [inline source/System.cpp:424]:
```suggestion
		static const string UNINHABITEDPLANET = "This planet doesn't have any place suitable for landing.";
```

File: source/System.cpp [hunk 2]
```diff
@@ -421,10 +421,10 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
...
 		static const string HOTMOON = "This moon is too hot to land on.";
 		static const string COLDMOON = "This moon is too cold to land on.";
-		static const string UNINHABITEDMOON = "This moon is uninhabited.";
+		static const string UNINHABITEDMOON = "This moon doesn't have any place suiatble for landing.";
```
Comment (bene-dictator) [inline source/System.cpp:427]:
```suggestion
		static const string UNINHABITEDMOON = "This moon doesn't have any place suitable for landing.";
```

**Review by bene-dictator — CHANGES_REQUESTED** [review]

**Review by quyykk — APPROVED** [review]
makes sense to me

File: source/System.cpp [hunk 3]
```diff
@@ -421,10 +421,10 @@ void System::Load(const DataNode &node, Set<Planet> &planets)
 		static const string STAR = "You cannot land on a star!";
 		static const string HOTPLANET = "This planet is too hot to land on.";
 		static const string COLDPLANET = "This planet is too cold to land on.";
-		static const string UNINHABITEDPLANET = "This planet is uninhabited.";
+		static const string UNINHABITEDPLANET = "This planet doesn't have any place suitable for landing.";
```
Comment (warp-core) [inline source/System.cpp:424]: Perhaps just remove the question of suitability and say there's no place you can land?
I prefer `anywhere` to `any place`, but that may be subjective.
```suggestion
		static const string UNINHABITEDPLANET = "This planet doesn't have anywhere you can land.";
```
```suggestion
		static const string UNINHABITEDPLANET = "This planet doesn't have any place you can land.";
```
Alternatively:
```suggestion
		static const string UNINHABITEDPLANET = "You can't land on this planet.";
```

Comment (bene-dictator) [reply source/System.cpp:424]: I think the first suggestion is clearer than both the PR and the second one; the second one doesn't give a reason and it looks like an arbitrary restriction.

**Review by RisingLeaf — APPROVED** [review]

**Review by bene-dictator — APPROVED** [review]

**Review by warp-core — APPROVED** [review]

**Pull request merged**